addon/services/scroller.js

```diff
@@ -59,7 +61,7 @@ export default Service.extend({
     let defaults = this.get('defaults');
     let axis = options.axis || defaults.axis;
     let ignoreViewport = typeof options.ignoreViewport !== 'undefined' ? options.ignoreViewport : defaults.ignoreViewport;
-    let container = options.container && this.getContainer(options.container) || window;
+    let container = options.container && this.getContainer(options.container) || this.get('window') || window;
```

Comment:
@kellyselden informed me that this will throw if the service doesn't exist. Adding a `getOwner` check shortly.